src/Button.js

```diff
@@ -199,12 +199,15 @@ const StyledButton = styled.button`
 
   i {
     vertical-align: middle;
+    color: inherit;
```

Comment:
I'm surprised this is necessary. I would have imagined this would be the default.

Reply:
Nope, I also don't think it inherits hover color unless we set it explicitly 😑